packages/devcmd-cli/src/index.ts

```diff
@@ -83,6 +84,6 @@ async function runInDevCmdsDir(dirPath: string): Promise<void> {
   try {
     await startProcess('node', ["-e", `require('devcmd/from-cli').run(...process.argv.slice(1))`, ...args], dirPath);
   } catch (err) {
-    isError(err) ? abort(err.message) : process.exit(1);
+    isError(err) ? abort(err.stack ?? err.message) : process.exit(1);
```

Comment:
Note: in most scenarios, `err` will be a status code here, since the Process Promise resolves/rejects with the status code on cloes. **IF** the process could not be instanciated (which is unlikely) the Promise is rejected with the occuring error. In that scenario, the stack trace of the error is printed if available.